src/Form.js

```diff
@@ -175,7 +182,7 @@ export default class Form extends InputContainer {
         });
     }
 
-    _validateOne(iptName, context) {
+    _validateOne(iptName, context, setError=true) {

```

Comment:
maybe call `isDisplayingError`, also make sure the argument in the function has proper spacing (ie `isDisplayingError = true` vs `isDisplayingError=true`